cmd/rancherd/package/.drone.yml

```diff
@@ -0,0 +1,78 @@
+---
+kind: pipeline
+name: amd64
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: tag
+  image: rancher/dapper:v0.4.2
+  commands:
+    - echo "${DRONE_TAG}-amd64" | sed -e 's/+/-/g' >.tags
+  volumes:
+    - name: docker
+      path: /var/run/docker.sock
+- name: docker-publish
+  image: plugins/docker
+  settings:
+    dockerfile: Dockerfile
+    build_args:
+      - ARCH=amd64
+      - TAG=${DRONE_TAG}
+    password:
+      from_secret: docker_password
+    repo: "rancher/rancherd-upgrade"
+    username:
+      from_secret: docker_username
+  when:
+    instance:
+    - drone-publish.rancher.io
+    ref:
+    - refs/head/master
+    - refs/tags/*
+    event:
+    - tag
+
+volumes:
+- name: docker
+  host:
+    path: /var/run/docker.sock
+
+
+---
+kind: pipeline
+name: manifest
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+  - name: manifest
```

Comment:
do you still need the manifest step? youre not doing it in the maine .drone.yml

Reply:
That was for if we support multiple architectures in rancherd-upgrade, since we dont, removed